src/component/CalculatorOperation.js

```diff
@@ -0,0 +1,40 @@
+import CalculatorOperationItem from "./CalculatorOperationItem";
+const OPERATIONS = ["/", "*", "-", "+", "="];
+
+const CalculatorOperation = ({ totalState }) => {
+  const onClickOperation = (operation) => {
+    if (
+      totalState.total === "0" ||
+      OPERATIONS.includes(totalState.total.slice(-1))
+    ) {
+      alert("숫자를 먼저 입력한 후 연산자를 입력해 주세요.");
+    } else {
+      totalState.setTotal(totalState.total + operation);
+    }
+
+    if (operation === "=") {
+      onClickReturn();
+    }
+  };
+
+  const onClickReturn = () => {
+    totalState.setTotal(Math.floor(Number(eval(totalState.total))));
+    if (eval(totalState.total) === Infinity) {
+      totalState.setTotal(() => "오류");
```

Comment:
```suggestion
      totalState.setTotal("오류");
```

콜백함수의 인자를 사용하지 않는다면 굳이 Dispatcher에 콜백함수를 넘길 필요는 없는것 같아요 😅

Reply:
useState의 비동기 특징 때문에 콜백함수로 넘기는게 습관이 되어버렸네요.. 수정하겠습니다!